src/core/css/SearchResult.scss

```diff
@@ -40,6 +49,44 @@
   }
 }
 
+.SearchResult-authors {
+  display: inline-block;
+  font-weight: normal;
+  margin: 0;
+  padding: 0;
+
+  &::after {
+    content: ' — ';
+    display: inline-block;
+    margin: 0 0.25rem;
+  }
+}
+
+.SearchResult-author,
+.SearchResult-users {
+  color: $secondary-color;
+  font-size: $font-size-default;
+}
+
+.SearchResult-author {
+  border-right: 1px solid #333;
```

Comment:
Is this separating the authors by a `|` essentially? Should we show more than one? The results boxes are pretty small. Maybe it would be better to show `Mark Striemer` (if I'm the only author) or just `3 authors` or `Mark Striemer and 2 others` or something actually good if there are more?

It looks like we're going to run out of space on that line quite quickly with a long author name or multiple authors and the icon and user count on the right.

I'm not sure if there are mocks for this view so maybe it's moot right now?